walkthrough.ipynb

```diff
@@ -0,0 +1,1950 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 🔎 Practical Walkthrough: Information Retrieval with pv211-utils\n",
+    "\n",
+    "Welcome to this walkthrough notebook for the [**pv211-utils**](https://github.com/MIR-MU/pv211-utils) — an open-source Python library that provides an object-oriented interface for building and evaluating information retrieval search engines for various text collections. It is designed to help you implement and experiment with concepts from PV211: Introduction to Information Retrieval course taught at the Faculty of Informatics, Masaryk University, Brno, Czech Republic. \n",
+    "\n",
+    "\n",
+    "Code implementations here are mostly linked to concepts from the book [Introduction to Information Retrieval](https://nlp.stanford.edu/IR-book/) by Manning, Raghavan, & Schütze (2008). You can download the full PDF version [HERE](https://nlp.stanford.edu/IR-book/pdf/irbookonlinereading.pdf).\n",
+    "\n",
+    "\n",
+    "- Starting off (cloning and setting up the repo) -> [Click here](#starting-off)\n",
```

Comment:
I am not sure why, but the links do not work when looking at the notebook in Github, i.e. here: https://github.com/MIR-MU/pv211-utils/blob/1ad51a5874f5f8ce6d7c44a5ed7db0a422f72bca/walkthrough.ipynb

maybe there would be some easy fix, but I understand that the intended use is probably to download and access the notebook from some jupyter env, right